scripts/ck3.ts

```
@@ -106,12 +180,13 @@ function parseLine (line: string) {
     return []
   }
 
-  const separatedLine = line.match(/(.*:)(\d*)( *)(".*")/)
+  const separatedLine = line.match(/(.*:)(\d*)( *)(".*")(?:#(.*))?/)
 
   if (separatedLine) {
     return [
       separatedLine[1].trim().replace(/:$/, ''),
       separatedLine[4].replace(/^"(.+)?"$/, '$1'),
+      separatedLine[5].trim(),
     ]
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)